index.html

```diff
@@ -14,52 +14,67 @@
   </head>
 
   <body>
-    <header>
+    <div class="container1">
+      <header>
+        <a class="logo" href="#">My Logo</a>
+
+        <nav>
+          <img class="dropdown" src="./images/Union.png" alt="Menu" />
+        </nav>
+      </header>
+    </div>
+
+    <div class="flex-container2">
       <a class="logo" href="#">My Logo</a>
 
-      <nav><img src="./images/Union.png" alt="Menu" /></nav>
-    </header>
-    <section class="background">
-      <h1>
-        I’m Peter. <br />
-        Glad to see <br />
-        you!
-      </h1>
-      <p>
-        I’m a software developer! I can help you <br />build a product , feature
-        or website Look <br />through some of my work and <br />experience! If
-        you like what you see and <br />have a project you need coded, don’t
-        <br />hestiate to contact me.
-      </p>
-      <div class="media">
-        <h3>LET'S CONNECT</h3>
-        <div class="socials">
-          <ul>
-            <li class="icon">
-              <a href="https://twitter.com/mwovi_peter"
-                ><img src="./images/twitter.png" alt="twitter link"
-              /></a>
-            </li>
-            <li class="icon">
-              <a href="https://www.linkedin.com/in/peter-mwovi-57141a179/"
-                ><img src="./images/Linkedin_icon.png" alt="linked link"
-              /></a>
-            </li>
-            <li class="icon">
-              <a href="#"><img src="./images/M.png" alt="M" /></a>
-            </li>
-            <li class="icon">
-              <a href="https://github.com/peter-mwovi/"
-                ><img src="./images/github.png" alt="github link"
-              /></a>
-            </li>
-            <li class="icon">
-              <a href="#"><img src="./images/doits.png" alt="doit" /></a>
-            </li>
-          </ul>
+      <nav>
+        <ul class="nav-links">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+    </div>
+
+    <div class="flex-container3">
+      <section class="background">
+        <h1>I’m Peter Mwovi. Glad to see you!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <div class="media">
+          <h3>LET'S CONNECT</h3>
+          <div class="socials">
+            <ul>
+              <li class="icon">
+                <a href="https://twitter.com/mwovi_peter"
+                  ><img src="./images/twitter.png" alt="twitter link"
+                /></a>
+              </li>
+              <li class="icon">
+                <a href="https://www.linkedin.com/in/peter-mwovi-57141a179/"
```

Comment:
- [ ] Ass mentioned by the previous reviewer please add a bit of padding at the bottom of the hero section to match with the provided figam design

|Figma design| Your design|
|---|---|
|<img width="1199"  alt="Screenshot 2023-06-10 at 3 06 03 PM" src="https://github.com/peter-mwovi/portfolio_setup/assets/44798044/6349394c-1508-47a0-a9f3-c017fd9bc72f">|<img width="1499" alt="Screenshot 2023-06-10 at 3 05 49 PM" src="https://github.com/peter-mwovi/portfolio_setup/assets/44798044/88e7ebdd-1ca2-48ba-b676-6537ec78fcde">|